guides/release/in-depth-topics/rendering-values.md

```diff
@@ -71,6 +71,25 @@ or via a property on some object
 ```handlebars
```

Comment:
If we are trying to equate this file to the JS backing it then we should specify the filename for both. There is information in the styleguide on how to do this https://github.com/ember-learn/guides-source/blob/master/CONTRIBUTING.md#style-guide